research/modeling/bert.py

```diff
@@ -0,0 +1,291 @@
+import tensorflow as tf
+import pandas as pd
+import tensorflow_hub as hub
+import os
+import re
+import numpy as np
+from bert.tokenization import FullTokenizer
```

Comment:
http://google.github.io/styleguide/pyguide.html#22-imports

import modules not classes